integration-tests/grpc/src/main/resources/application.properties

```
@@ -15,7 +15,7 @@
 ## limitations under the License.
 ## ---------------------------------------------------------------------------
 
-quarkus.native.resources.includes=certs/*.key,certs/*.pem,keys/*.json
+quarkus.native.resources.includes=target/certs/*.key,target/certs/*.pem,keys/*.json
```

Since certs are now read from the filesystem, I think this can be reduced to:

```suggestion
quarkus.native.resources.includes=keys/*.json
```

---

fixed